src/js/components/Select/doc.js

```diff
@@ -26,6 +26,23 @@ export const doc = Select => {
     closeOnChange: PropTypes.bool
       .description('Wether to close the drop when a selection is made.')
       .defaultValue(true),
+    clear: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({
+        position: PropTypes.oneOf(['top', 'bottom']).description(
+            `Add a clear value to the top or at the bottom of the
+            container. By default no unselect option is present.`,
+        ),
+        label: PropTypes.string
+          .description('Label for the clear selection item')
+          .defaultValue('Clear selection'),
```

Comment:
how does it show that `label` is a boolean? If it is we'll need to update index.d.ts

Reply:
shouldn't be a boolean. its just non mandatory. not sure what you mean